web/ui/src/Pools/edit-pool.html

```diff
@@ -0,0 +1,16 @@
+<div class="form-group">
+  <label for="pool_id">
+      <span translate>label_pool_name</span>:
+  </label>
+  <input readonly class="form-control" id="pool_id" type="text" ng-model="editablePool.ID">
+</div>
+
+<div class="form-group">
+  <label for="edit_pool_timeout">
+      <span translate>label_connection_timeout</span> (eg 2s or 1m):
```

Comment:
This can demonstrate the varying units as well as the fact it can take a bunch of durations at once with something like "1h 3m 30s"